src/views/createCommunity/index.tsx

```diff
@@ -1331,7 +1334,7 @@ function CreateCommunityScreen() {
                                 </View>
                                 {coverImage.length > 0 ? (
```

Comment:
A few lines below, you've changed `uri: coverImage` to `uri: userCommunity?.cover?.url`. But here you didn't. It's confusing.

Reply:
Actually, need to be coverImage since coverImage is a state we use either to initialize with the community current image or with the new one. Fixed.